src/OrchardCore.Modules/OrchardCore.Users.TimeZone/OrchardCore.Users.TimeZone.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.ContentManagement.Display\OrchardCore.ContentManagement.Display.csproj" />
```

Comment:
Try to remove these references and only add the ones that are not already imported automatically.

Reply:
fixed